### ISSUE TITLE
Apple Script support?

### ISSUE THREAD

**Issue by linxichen** [monoclesociety/eggscellent-classic#4]
I am wondering if Eggscellent supports Applescript? If yes, will any documentation be released?

**Comment by devon** [comment]
Yes, Eggscellent support AppleScript. You may look at the source code.

**Comment by linxichen** [comment]
Hi Devon,

Thanks for the reply. But I am kinda newbie. Can you be more specific and point me to the exact source files?

**Comment by devon** [comment]
Mac OS X - pomodorable - Resources - AppleScripts
